uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -80,20 +87,25 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
   @Before
   def beforeEach(): Unit = {
     intelliJ.probe.openProject(repositoryMainDir)
-    intelliJ.machete.configureProject()
+    intelliJ.project.configure()
     intelliJ.probe.awaitIdle()
+
+    // Since 2021.1, IntelliJ introduced a mechanism of trusted projects... yet another dialog to accept in our case :/
```

Comment:
Quite a major change... and apparently only introduced in the recent beta (earlier EAPs of 202.1 didn't get stuck on this dialog) lol